nectar/config.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from builtins import object
```

Comment:
Sorry for late reply. I was on holiday and I am quite busy right now. I think that this PR can be closed because of your plans to retire this project as was mentioned in other PR here: https://github.com/pulp/nectar/pull/50#issuecomment-247060388